src/doc/tarpl/ownership.md

```diff
@@ -0,0 +1,66 @@
+% Ownership and Lifetimes
+
+Ownership is the breakout feature of Rust. It allows Rust to be completely
+memory-safe and efficient, while avoiding garbage collection. Before getting
+into the ownership system in detail, we will consider the motivation of this
+design.
+
+We will assume that you accept that garbage collection is not always an optimal
```

Comment:
I recommend adding "(GC)" after "garbage collection" since you write "GC" below.